bindings/rust/s2n-tls-sys/README.md

```diff
@@ -1,3 +1,30 @@
 This crates provides low level rust bindings for [s2n-tls](https://github.com/aws/s2n-tls) which are autogenerated with [bindgen](https://github.com/rust-lang/rust-bindgen)
 
-This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
\ No newline at end of file
+This crate is not intended for direct consumption by end consumers. Interested developers should instead look at the [s2n-tls](https://crates.io/crates/s2n-tls) or [s2n-tls-tokio](https://crates.io/crates/s2n-tls-tokio) crates. These  provide higher-level, more ergonomic bindings than the `s2n-tls-sys` crate.
+
+This crate supports using pre-built libs2n following the next steps below:
+
+1. Clone [s2n-tls](https://github.com/aws/s2n-tls) and compile your preferred configuration of s2n-tls. 
+
+You may choose to link against a specific libcrypto at this step. For more information, see [Building with a specific libcrypto](https://github.com/aws/s2n-tls/blob/main/docs/BUILD.md#building-with-a-specific-libcrypto)
+```
+cmake . -Bbuild -DBUILD_SHARED_LIBS=on -DBUILD_TESTING=off
+cmake --build build -- -j $(nproc)
+```
+
+2. cd into your rust project and set environment variables to libs2n library sources. 
+
+This tells the bindings to link to pre-built libs2n when running the build script for s2n-tls-sys
```

Comment:
Let's include the raw definitions, because I think your example includes some assumptions which might not actually match customer environments.

E.g. call out that `S2N_TLS_LIB_DIR` points to the folder containing the `libs2n.a` artifact that you would like `s2n-tls-sys` to link against. 

Also, maybe let's call out whether something is different with static vs dynamic libs?

Reply:
> maybe let's call out whether something is different with static vs dynamic libs?
I understand static lib bakes the code to executable at build time and dynamic lib requires the linked libraries to be available at runtime. Is there something else we want to call out within the context of this bindings generation?